utilities/merge_operators/cassandra/merge_operator.h

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2017-present, Facebook, Inc.  All rights reserved.
+// This source code is licensed under the BSD-style license found in the
+// LICENSE file in the root directory of this source tree. An additional grant
+// of patent rights can be found in the PATENTS file in the same directory.
```

Comment:
Add the couple of lines that were added in #2226, for GPLv2, for all the new files.